packages/wonder-blocks-banner/src/components/banner.js

```diff
@@ -68,6 +68,12 @@ type BannerValues = {|
 |};
 
 type Props = {|
+    /**
+     * Accessible label for the banner.
+     * This is read out before the other contents of the banner.
+     */
+    ariaLabel?: string,
```

Comment:
**suggestion:** Some of our other components use `aria-label` so we should probably be consistent with those.